### ISSUE TITLE
Instalation fails with SQLSTATE[08006] [7] FATAL:  password authentication failed for user "default" for PostgreSQL container

### ISSUE THREAD

**Issue by jonathands** [EscolaLMS/Create-LMS-App#46]
The relevant logs for the postgres container follow

2024-03-18 19:08:41.667 CET [1] LOG:  starting PostgreSQL 12.16 (Debian 12.16-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
2024-03-18 19:08:41.675 CET [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2024-03-18 19:08:41.675 CET [1] LOG:  listening on IPv6 address "::", port 5432
2024-03-18 19:08:41.680 CET [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2024-03-18 19:08:41.726 CET [27] LOG:  database system was shut down at 2024-03-18 19:08:29 CET
2024-03-18 19:08:41.732 CET [1] LOG:  database system is ready to accept connections
2024-03-18 19:09:11.740 CET [41] FATAL:  role "root" does not exist
2024-03-18 19:09:14.470 CET [42] FATAL:  password authentication failed for user "default"
2024-03-18 19:09:14.470 CET [42] DETAIL:  Password does not match for user "default".
	Connection matched pg_hba.conf line 99: "host all all all md5"
2024-03-18 19:09:14.471 CET [43] FATAL:  password authentication failed for user "default"
2024-03-18 19:09:14.471 CET [43] DETAIL:  Password does not match for user "default".
	Connection matched pg_hba.conf line 99: "host all all all md5"
	
	
Running  docker exec -it <PG-CONTAINER-HASH>  psql -U default -p 5432 will connect and the default database is empty

**Comment by qunabu** [comment]
Have you run `make init` after it was init already ? 

Can you try `make refresh` - but please beware it will remove data from your current database 

Reason why

`make init` creates a random secure credential for postgres database, then pass this as [docker compose environmental variables](https://hub.docker.com/_/postgres) and also persist data in the `docker/postgres-data` folder 

when folder `docker/postgres-data` exists then changing credentials as environmental variables won't work 

in other words you should not run `make init` if `docker/postgres-data` folder exists
